src/app/components/tile/index.html

```diff
@@ -214,9 +214,11 @@
     <p>
       Tiles are containers to organize varied content. They divide pages into multiple categories and group related content. Use tiles to provide access to a wide range of content where users benefit from the ability to reorder, expand and collapse, and personalize tiles.
     </p>
-
+    <p>
+      The content within tiles can vary widely, but each tile follows <a routerLink="/design/principles">SKY UX design principles</a> and <a routerLink="/design/guidelines">design guidelines</a> for that typography, layout, and other design considerations.
```

Comment:
last part of the sentence reads a bit awkwardly, maybe something like "for layout, typography and other design consuderations."

Reply:
I agree with Todd, but I don't really have a preference for order of items. I'd just eliminate the word "that" before typography.

Reply:
Yep. `that` definitely shouldn't be there. It's just a leftover word that didn't get deleted when I was tweaking the sentence. I'll remove it and flip the order like Todd suggests.